### ISSUE TITLE
[Nukkit] LuckPerms won't load Permission data when using Nemisys

### ISSUE THREAD

**Issue by funniray** [lucko/LuckPerms#946]
Whenever a player gets transferred to another server using Nemisys (The official proxy server for Nukkit), I get this error
```
16:59:06 [INFO] [LuckPerms] [WARN] User 3ff3f8a6-e264-3586-b633-9a86993bbc45 - funniray doesn't have data pre-loaded. - denying login.
16:59:06 [INFO] funniray[/192.168.254.28:49180] logged out due to [LP] Permissions data could not be loaded. Please try again later.
```
I'm using LuckPerms v4.0.141 and Nukkit build 145.

**Comment by BrainStone** [comment]
Did you try the latest LP build?

**Comment by funniray** [comment]
Updated to LuckPerms v4.1.82 and same issue

**Comment by lucko** [comment]
Seems like the pre-login event is being called after the login event.

Yes, this is definitely a Nukkit/Nemisys issue, you should report it to them.

**Comment by funniray** [comment]
It was fixed in the latest commit of SynapseAPI, faf298ced5ddb3fa55592a30122c6d62c4969abd
